linux/net/rina/rmt.h

```diff
@@ -59,6 +59,7 @@ struct rmt;
 enum flow_state {
 	N1_PORT_STATE_ENABLED,
 	N1_PORT_STATE_DISABLED,
+	N1_PORT_STATE_DO_NOT_DISABLE,
```

Comment:
All I mean is that it looks to me that this PR is abusing the flow_state enum to share a variable between some functions. The flow_state is not (or at least should not be) implementation specific, it should be defined independently of the implementation and reflect the state machine for the flow/port. N1_PORT_STATE_DO_NOT_DISABLE does not seems like something that's fundamental for the FSM of a flow. For the solution to this issue, you can, for instance, define a variable outside of the flow_state enum, that's all.